uber/automated_emails.py

```diff
@@ -14,14 +14,16 @@ class AutomatedEmail:
     }
     extra_models = queries  # we've renamed "extra_models" to "queries" but are temporarily keeping the old name for backwards-compatibility
 
-    def __init__(self, model, subject, template, filter, *, sender=None, extra_data=None, cc=None, bcc=None, post_con=False, needs_approval=True):
+    def __init__(self, model, subject, template, filter, *, sender=None, extra_data=None, cc=None, bcc=None, post_con=False, needs_approval=True, email_id=None):
         self.model, self.template, self.needs_approval = model, template, needs_approval
         self.subject = subject.format(EVENT_NAME=c.EVENT_NAME)
         self.cc = cc or []
         self.bcc = bcc or []
         self.extra_data = extra_data or {}
         self.sender = sender or c.REGDESK_EMAIL
         self.instances[self.subject] = self
+        self.email_id = email_id or self.subject
+        self.email_id = self.email_id.format(EVENT_NAME=c.EVENT_NAME)
```

Comment:
I left this essentially as-is (though I did clean it up, as suggested) because the idea behind it is that you can change an email's subject by copying the old subject into the `ident` variable. We may want to undo the formatting later for the reason you gave, but since MAGLabs is up and running and then MAG2017 will be using the old format, it'll be a while.